src/main/resources/application.conf

```diff
@@ -2,3 +2,14 @@ akka {
   loglevel = INFO
   log-dead-letters-during-shutdown = off
 }
+
+# The server pipeline configuration to capture call metrics.
+squbs.pipeline.server.default {
+  pre-flow = sampleMetricsFlow
```

Comment:
We should set this only for `pre-flow`.  Otherwise, it will install the metrics flow both to the beginning and to the end of the flow.

Also, I would not name it as `sampleMetricsFlow`.  It is actually not sample, it is real metrics utility provided by squbs.  In addition, for now it may be just metrics, but we may add further stuff in the future.  

I think `defaultFlow` would make sense, but you may come up with a better one.  If you do not like this, I would also be ok with just `metricsFlow`.